mesos-docker/run/run.sh

```diff
@@ -198,6 +206,31 @@ function start_slaves {
       HADOOP_VOLUME=
     fi
 
+
+    if [[ -n $SLAVES_CONFIG_FILE ]]; then
+      if [[ -f $SLAVES_CONFIG_FILE ]]; then
```

Comment:
The check to see if the file exists should be done during arguments parsing/checking.
It is always annoying for the user to get half-way through a process, and then get a error because a provided argument is pointing to nothing.